content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +85,142 @@ func TestAccXxx(*testing.T)
 
 ## Running Acceptance Tests
 
-A `terraform` binary of version 0.12.26 or above must be available on the local system for the acceptance test driver to run, either on the system `$PATH` or supplied via the `TF_ACC_TERRAFORM_PATH` environment variable described below. If you are running acceptance tests in a CI environment, we recommend that you download and install Terraform with a prior build step.
+~> **NOTE**: Acceptance tests typically create and destroy actual infrastructure resources, possibly incurring expenses during or after the test duration, and often require network connectivity and provider-specific credentials.
 
-Terraform requires an environment variable `TF_ACC` be set in order to run
-acceptance tests. This is by design, and intended to prevent developers from
-incurring unintended charges when running tests. The easiest way to run
-acceptance tests is to use the built in `make` step `testacc`, which explicitly
-sets the `TF_ACC=true` value for you. Example:
+Acceptance testing runs Terraform CLI commands to end-to-end verify provider functionality. Terraform CLI version 0.12.26 or later is required. Creating real infrastructure in tests verifies the described behavior of providers in real world use
+cases against the actual APIs, and verifies both local and remote state values match. It is recommended to run acceptance tests in a separate testing account or namespace. This ensures no unexpected infrastructure is modified or destroyed due to potential code or testing issues.
```

Comment:
I think it could make sense to move the version info and the recommendation about namespaces to a new section below....refer to my comment there :-) 

```suggestion
Acceptance testing runs Terraform CLI commands to verify provider functionality end to end.  Creating real infrastructure in tests verifies the behavior of providers in real-world use cases and verifies that local and remote state values match. 
```